packages/boa/lib/index.js

```diff
@@ -80,7 +80,10 @@ function copy(T) {
 function dump(T) {
   return pyInst.import('json')
     .__getattr__('dumps')
-    .invoke(asHandleObject(T));
+    .invoke(asHandleObject(T), {
+      default: _internalWrap(builtins).str,
```

Comment:
Is this a typo? It prints builtins when dump T?

Reply:
default is the function that will be used when some fields cannot be serialized. By default, we use str method as this method

Reply:
I see, may I ask you to add a comment there to explain?

Reply:
Sure, done